### DISCUSSION TITLE
Cloning The Repository v2.0.0

### DISCUSSION THREAD

**Discussion by raj-brown** [E3SM-Project/E3SM#4737] [E3SM model help]
Dear All-
I am trying to clone v2.0.0 with the submodules and It always failed. I tried forking the repo, directly cloning it, but no success. It clones to certain extent and throw the error that please check right permission. Could you please help me in this regard. I am loosing a lot of time in just cloning the repo. Also, all the webpages pointing to help on E3SM  web is dead. 
I will greatly appreciate if your help.

Thanks!
regards
Raj

**Comment by jonbob** [comment]
Hi Raj,
I'm afraid you'll have to give us more information about exactly what you did, what platform you were on, and a bit more about the errors you saw.

Thanks - Jon

**Reply by raj-brown** [reply]
@jonbob @rljacob  Hi Jon and Jacob- I am cloning it on my DGX box with Ubuntu 20.04.
I have already added the ssh keys to github.

Thanks!

**Comment by rljacob** [comment]
permission errors usually mea you need to add ssh keys to github.    Or edit .gitmodules and change all the urls from git to http

**Comment by jonbob** [comment]
It would still be useful if you post the commands you've run and the errors you see
